src/transformers/models/llava_next/image_processing_llava_next.py

```diff
@@ -603,6 +604,13 @@ def preprocess(
             pixel_values = np.array(pixel_values)
             new_images.append(pixel_values)
 
+        max_patch = max(len(x) for x in new_images)
```

Comment:
I am running into a problem here; when we use a combination of a streaming dataset, processor and  a collator function during training, the different processed 'pixel_values' being collated can have different number of num_patches, which prevents them from being stacked into a tensor. Providing padding as a separate utility should solve that problem too I believe.